src/component/Modal/ModalContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from "react";
+import { createPortal } from "react-dom";
+import { useRecoilValue } from "recoil";
+
+import { default as modalState } from "@recoil/modalList";
+import useModal from "@hooks/useModal";
+import * as S from "./style";
+import * as Modal from "./ModalList";
+
+const ModalContainer = () => {
+  const modalList = useRecoilValue(modalState);
+  const { hideModal } = useModal();
+
+  useEffect(() => {
+    if (modalList.length === 0) {
+      document.body.style.overflow = "auto";
+    } else {
+      document.body.style.overflow = "hidden";
+    }
+    return () => {
+      if (modalList.length === 0) {
+        document.body.style.overflow = "auto";
+      } else {
+        document.body.style.overflow = "hidden";
+      }
+    };
+  }, [modalList]);
```

Comment:
```suggestion
  useEffect(() => {
    const containerStyle = document.body.style;
    
    if (modalList.length === 0) containerStyle.overflow = "auto";
    else containerStyle.overflow = "hidden";
    return () => {
      if (modalList.length === 0) containerStyle.overflow = "auto";
      else containerStyle.overflow = "hidden";
    };
  }, [modalList]);
```
이것도 저는 개인적으로는 이렇게 할 거 같습니다~~~~~
뭐 중괄호가 많고 적고에 대해서 논하는 거는 개인적인 차이가 있을테니 크게 신경 안쓰셔도 되는 리뷰입니다~~~~

Reply:
변수로 할당하는것도 괜찮네요 근데 얘도 그냥 삼항으로 해야겠다 ㅋㅋ
`document.body.style.overflow = modalList.length === 0 ? "auto" : "hidden";`